File: template/scalaServlet/src/main/scala/com/app/controller/MainController.scala
package com.app.controller

import com.app.view.Layout

class MainController extends Controller {
	
  val ROUTE = "/"

	override def handles(route: String): Boolean = route.equals(ROUTE)

	override def execute(context: Context) {
		context.response.setContentType("text/html")
		context.response.setCharacterEncoding("UTF-8")

		context.response.getWriter.write(new Layout().build().render())
  }
}

File: template/servlet/src/main/java/com/app/controller/MainController.java
package com.app.controller;

import com.app.view.Layout;

public class MainController implements Controller {

	@Override
	public boolean handles(String route) {
		return "/".equals(route);
	}

	@Override
	public void execute(Context context) throws Exception {
		context.response().setContentType("text/html");
		context.response().setCharacterEncoding("UTF-8");

		context.response().getWriter().write(new Layout().build().render());
	}
}
